README.md

```diff
@@ -185,8 +185,8 @@ Since dumb-init is released as a statically-linked binary, you can usually just
 plop it into your images. Here's an example of doing that in a Dockerfile:
 
 ```Dockerfile
-RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64
-RUN chmod +x /usr/local/bin/dumb-init
+RUN wget -O /usr/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64
+RUN chmod +x /usr/bin/dumb-init
```

Comment:
the rest of the README uses `/usr/bin/dumb-init` as that's where OS package managers will put files.  non-package-manager-managed files should go in the `/usr/local` prefix

(so, despite it looking weird -- it's correct on master)

Reply:
Got it. If you don't believe a note about it in README, we can close this. Thanks. 😃